packages/opentelemetry-sdk-trace-base/src/Span.ts

```diff
@@ -191,10 +191,12 @@ export class Span implements api.Span, ReadableSpan {
 
     if (this._duration[0] < 0) {
       api.diag.warn(
-        'Inconsistent start and end time, startTime > endTime',
+        'Inconsistent start and end time, startTime > endTime. Setting end time 1ms after start time.',
```

Comment:
really 1ms? Can't see this in code and I don't think we should create 1ms spans in this case.

Reply:
Ah sorry that wording is from a previous iteration. i changed it to 0ms